source/extensions/transport_sockets/tls/context_impl.cc

```diff
@@ -782,11 +788,21 @@ size_t ContextImpl::daysUntilFirstCertExpires() const {
   return daysUntilExpiration;
 }
 
+uint64_t ContextImpl::secondsUntilFirstOcspResponseExpires() const {
```

Comment:
How do gauges work when there are no ocsp responses? 0 feels like the only reasonable base case and also confusing